### ISSUE TITLE
Inconsistent UI

### ISSUE THREAD

**Issue by Foggalong** [the8472/unicode-keyboard#1]
On some websites the full user interface appears (e.g. GitHub issues)
![screenshot_2017-08-31_16-25-59](https://user-images.githubusercontent.com/5920259/29931422-2583b9d0-8e69-11e7-8b83-5da14e02c4b8.png)

On some only the search bar appears, which makes it difficult to use (e.g. Reddit comments)
![screenshot_2017-08-31_16-28-51](https://user-images.githubusercontent.com/5920259/29931557-84ce2560-8e69-11e7-9564-098e62abbd8d.png)

And on some it doesn't appear at all (e.g. search on addons.mozilla.org)
![screenshot_2017-08-31_16-26-52](https://user-images.githubusercontent.com/5920259/29931508-61ff5de2-8e69-11e7-96c8-055a23716a95.png)

Happy to provide any additional information needed to help solve this issue

**Comment by the8472** [comment]
Yeah. Currently I embed an iframe into the top level frame of the web page. This is fragile because it is subject to many different ways the root element of the page can be laid out.

I'll have to experiment with the [window.create](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/windows/create) API to see if I can create a decent panel below the page. Or maybe the [devtools](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/devtools.panels) since the sidebar API does not provide bottom placement.

**Comment by the8472** [comment]
I have bugged the mozilla devs, currently there is no API that provides what is needed in a reliable way. It has to wait until [bug 1340930](https://bugzilla.mozilla.org/show_bug.cgi?id=1340930) is implemented.

**Comment by ceving** [comment]
I think you should use the sidebar instead. [PwdHash][1] for  example uses the sidebar.

[1]: https://github.com/quassy/pwdhash
